### PR TITLE
Add files to set up project

### PR DESCRIPTION
**Pull request by hendridg**
## Set up project

In this milestone we:

- Added index.html file, markup home page.
- Added style.css, basis styles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
-# microverse-week-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> One paragraph statement about the project.
```
Comment (rotimiazeez) [inline README.md]: - [ ] Kindly add a project summary to briefly explain what was done in the project😊

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+
+![screenshot](./app_screenshot.png)
```
Comment (rotimiazeez) [inline README.md:7]: - [ ] Kindly fix the image link it's broken😢👍

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @hendridg 👋, 

### Good job so far!👏
### Summay 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
### There are some issues that you still need to work on to go to the next project but you are almost there!✌
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Getting Started
```
Comment (andersonlebon) [inline README.md:17]: - [ ] In the README the section `Getting started` should explain how someone else can run or get the project in his/her local environment. you can include something like:
```
To get the content of this project locally you need to run this command in your terminal:
- `git clone yourlink
- cd projectName
```

File: README.md [hunk 4]
```diff
@@ -1 +1,62 @@
...
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
```
Comment (andersonlebon) [inline README.md:36]: - [ ] Kindly remove all unused sections in your `README`. This will make it clear

File: README.md [hunk 5]
```diff
@@ -1 +1,62 @@
...
+
+👤 **Author1**
+
+- GitHub: [@hendridg](https://github.com/hendridg)
+- Twitter: [@hendridg](https://twitter.com/hendridg)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hendridg)
```
Comment (andersonlebon) [inline README.md:42]: - [ ] I can see you are still using the link of the README TEMPLATE. I think you forgot to put your own information. Kindly go ahead and put your valid links

File: README.md [hunk 6]
```diff
@@ -1 +1,62 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (andersonlebon) [inline README.md:62]: - [ ] All of these sections should be valid, I mean they should contain your own information.

**Review by andersonlebon — APPROVED** [review]
Hi @hendridg 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 6 above)
Comment (hendridg) [reply README.md:62]: Thank you @andersonlebon, you really help me a lot

**Pull request merged**